app/views/resources/edit.html.erb

```diff
@@ -1,3 +1,5 @@
 <%= render partial: "partials/form_structure", locals: {model_name: "resource",
                                                         object: @resource,
                                                         action: "edit"} %>
+
+<%= link_to "Return to this competency's resources ", resources_path(:competency_id => @competency_id), class: "waves-effect waves-light btn"  %>

```

Comment:
Standardize button to be large? That's how I have it in the Questions model. Can be changed.